### DISCUSSION TITLE
Long term support

### DISCUSSION THREAD

**Discussion by blujedis** [csandman/chakra-react-select#107] [Q&A] [answered]
Please don't take offense :) It looks as though commits are regular etc. On a scale of 1-10 can you give me an idea as to commitment? Have a large project I'm considering using this lib in. Happy to contrib etc. Just wondering if author is currently using in production. Honestly no wrong answer here. Thanks in advance.

**Comment by csandman** [comment, marked as answer]
Yes, I am currently using it in production!  Like I mentioned in the README, I have 2 relatively big changes coming out soon, updating to react-select v5 and adding more support for customizing the styles.  The only slowdown point I've had is my lack of experience with TypeScript.

At the end of the day, this is only a wrapper for react-select, so the main LTS I'd provide is keeping up to date with react-select and Chakra UI's versions.  And I do plan to continue doing that for the foreseeable future.  This is my first open source package thats really received any attention so I'd like to make it as good as I can.

On top of that, I'd definitely take the time to consider any PRs or requests people have, the more the community gets involved, the more I care about it!  The main thing I'm looking for help on right now is [upgrading to react-select v5](https://react-select.com/upgrade), (removing `@types/react-select`, adding the generics for react-select's types, etc.). I will get to this soon if no one submits a PR, but if not I just need a solid afternoon to figure it all out.  So if you have any TypeScript experience and want to give it a shot, be my guest!

**Comment by blujedis** [comment]
Thank you. I'm fairly solid with Typescript actually. It's all we do. I won't be able to give it real attention until roughly early December, but glad to assist there not a prob :) Thanks for the honest response that's all I was looking for!

**Comment by csandman** [comment]
Sounds good, appreciate the offer!  If you get a chance to look at the source and have any questions, feel free to ask!

**Comment by blujedis** [comment]
Already have it work easy peasy. I'll for sure look at the types so if you have anything to add on that, please add and I'll at least try to rough things out here over the coming days as I'm going to roll it out on several screens. But the first one was quite easy.

**Comment by csandman** [comment]
@blujedis Just a heads up, I implemented both of the major changes I had mentioned in the readme originally, adding a styling system in v2, and updating to react-select@5 in v3. As far as LTS goes, I'd say this package now has pretty much everything it needs to survive, with most changes from now on being related to keeping the dependencies up to date.  I will of course still consider any feature requests anyone has that are in the scope of this package, but besides that there isn't anything that I still have planned to add besides the couple things left in the [Roadmap](https://github.com/csandman/chakra-react-select#roadmap).

If you want to give v3 of this package a try when you get a chance, I'd be glad to hear any feedback!  Especially if anything doesn't work right.
